core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -315,16 +316,17 @@ class MasterSuite extends SparkFunSuite
     val conf = new SparkConf()
     val localCluster = new LocalSparkCluster(2, 2, 512, conf)
     localCluster.start()
+    val masterUrl = s"http://localhost:${localCluster.masterWebUIPort}"
     try {
       eventually(timeout(5.seconds), interval(100.milliseconds)) {
-        val json = Source.fromURL(s"http://localhost:${localCluster.masterWebUIPort}/json")
-          .getLines().mkString("\n")
+        val json = Utils
```

Comment:
Not easy to maintain the same format as the master, so added `masterUrl(line 319) `, any other suggestions @HyukjinKwon

Reply:
That's fine